deploy/uhc_cluster_role_binding.yaml

```diff
@@ -4,11 +4,11 @@ metadata:
   name: aws-account-operator-client
 subjects:
 - kind: ServiceAccount
-  namespace: aws-account-operator-client
+  namespace: aws-account-operator
```

Comment:
Note to reviewers: This bug was only a problem for local testing. It is not causing chaos in real deployments because CRBs are generated by OLM from the CSV, which [only includes the service account *name*](https://github.com/openshift/aws-account-operator/blob/17d1683aedd534af2f974326f662b8f7423a3bfb/boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py#L241-L249).